edn_format/edn_dump.py

```diff
@@ -157,7 +157,7 @@ def udump(obj,
             return indent_lines(lines, '[', ']', indent, indent_step)
     elif isinstance(obj, (set, frozenset)):
         if sort_sets:
-            obj = sorted(obj)
+            obj = sorted(obj, key=lambda v: str(v))
```

Comment:
This breaks on numbers: 99 should come before 100 but `str(100)` comes before `str(99)`. Note we probably have the same issue on dictionaries since we use `str` as well.